app/services/katello/product_content_finder.rb

```diff
@@ -18,7 +18,7 @@ def product_content
       if match_environment
         environment = consumable.lifecycle_environment
         view = consumable.content_view
-        return [] unless environment && view
+        return ProductContent.where("1=0") unless environment && view
```

Comment:
Ah, this is where we were returning the blank array :smile: 

`"1=0"` works fine, but it's always been just super weird to me.  What about `ProductContent.none` ?  That should always return an `ActiveRecord::Relation` as well.

Reply:
That sounds interesting :).. Lemme see